CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,41 @@
+Contributor Covenant Code of Conduct
```

Comment:
Ah whoops, missed that. Mind just fixing [the formatting](https://github.com/jdalton/underscore/blob/coc/CODE_OF_CONDUCT.md)